[21Aug2024 01:05:47.926] [Netty Epoll Server IO #0/WARN] [io.netty.channel.DefaultChannelPipeline/]: An exceptionCaught() event was fired, and it reached at the tail of the pipeline. It usually means the last handler in the pipeline did not handle the exception.
io.netty.channel.unix.Errors$NativeIoException: accept(..) failed: Software caused connection abort
[21Aug2024 01:51:21.919] [Server thread/INFO] [net.minecraft.server.network.ServerLoginPacketListenerImpl/]: /154.213.187.7:1 lost connection: Internal Exception: io.netty.handler.codec.DecoderException: java.io.IOException: Packet login/serverbound/minecraft:hello (ServerboundHelloPacket) was larger than I expected, found 1 bytes extra whilst reading packet serverbound/minecraft:hello
